imageregistry/v1/01-crd.yaml

```diff
@@ -7,6 +7,8 @@ spec:
   scope: Cluster
   version: v1
   preserveUnknownFields: false
+  subresources:
+    status: {}
```

Comment:
@sttts @soltysh would adding the `+kubebuilder:subresource:status` tag to the object generate this for us automatically?

Reply:
We're not running full generation, so this won't help, but it worthy adding in the nearest future.